emails/utils.py

```diff
@@ -406,10 +407,11 @@ def remove_trackers(html_content, level='general'):
     general_detail = count_tracker(html_content, GENERAL_TRACKERS)
     strict_detail = count_tracker(html_content, STRICT_TRACKERS)
     
-    for tracker in trackers:
-        pattern = convert_domains_to_regex_patterns(tracker)
-        changed_content, matched = re.subn(pattern, f'\g<1>{settings.SITE_ORIGIN}/faq\g<1>', changed_content)
-        tracker_removed += matched
+    if sample_is_active('foxfood-email-sample'):
```

Comment:
There's a difference between "I tried but didn't remove any trackers" and "My coin filp came out tails so I didn't remove any trackers". That means that we need to log the value of `sample_is_active('foxfood-email-sample')`, and pass it all the way to the `email_tracker_study_link` in `_sns_message`